### PR TITLE
Changes for adding retry delay

### PR DESCRIPTION
**Pull request by maheshrajamani**
**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [x] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/QueryExecutor.java [hunk 1]
```diff
@@ -164,26 +166,40 @@ private Uni<AsyncResultSet> executeSchemaChange(SimpleStatement boundStatement,
...
-                                  .setIdempotent(true)
-                                  .setSerialConsistencyLevel(
-                                      operationsConfig
-                                          .queriesConfig()
-                                          .consistency()
-                                          .schemaChanges())));
+                  .item(throwable)
+                  .onItem()
+                  .delayIt()
+                  .by(Duration.ofSeconds(1))
```
Comment (tatu-at-datastax) [inline src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/QueryExecutor.java:182]: Would it make sense to make this configurable? (and if so, specify in milliseconds, not seconds, to allow things like `200` msecs)

Comment (maheshrajamani) [reply src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/QueryExecutor.java]: Made it configurable. But from what we saw this needs delay in seconds.

Comment (tatu-at-datastax) [reply src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/QueryExecutor.java]: I am not sure I follow... why in seconds? I mean, default value being 1 second is probably fine and makes sense, but is it never useful to configure it in fractions (1.5 seconds etc)?

**Review by tatu-at-datastax — APPROVED** [review]

File: src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/QueryExecutor.java (hunk 1 above)
Comment (maheshrajamani) [reply src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/QueryExecutor.java]: Changed to millis as suggested.

**Review by jeffreyscarpenter — APPROVED** [review]

**Pull request merged**